electionleaflets/templates/people/person.html

```diff
@@ -3,31 +3,14 @@
 {% block content %}
   <div class="row">
     <div class="columns large-8">
-      <h1>{{ object.name }} – {{ object.current_party.party.party_name }}</h1>
-      <ul>
-        <li>
-          <a href="https://yournextmp.com/person/{{object.remote_id}}/{{ object.name|slugify }}">
-            Find out more about {{ object.name }} at YourNextMP.com
-          </a>
-        </li>
-        <li>
-          <a href="{% url "constituency-view" object.current_constituency.pk object.current_constituency.name|slugify %}">
-            See more leaflets sent in {{ object.current_constituency.name }}
-          </a>
-        </li>
-        {% if object.current_party %}
-        <li>
-          <a href="{% url "party-view" object.current_party.party.pk object.current_party.party.party_name|slugify %}">
-            See more leaflets sent by {{ object.current_party.party.party_name }}
-          </a>
-        </li>
-        {% endif %}
-      </ul>
+      <h1>{{ object.name }}</h1>
+      <a href="https://candidates.democracyclub.org.uk/person/{{object.remote_id}}">
```

Comment:
Genuine question: do you think YNR or WCIVF is the better link here?

Reply:
Good question, I think having had another look, WCIVF would be a better link.

Reply:
Updated :+1: